.github/workflows/main.yml

```diff
@@ -116,8 +116,13 @@ jobs:
           path: |
             ./apm-agent-plugin-sdk/target/apm-agent-plugin-sdk-*.jar
             !./**/*-sources.jar
-
-
+      - name: Upload benchmark binaries as artifacts
```

Comment:
i somehow messed up when running the git merge and this is here while it should not... since it was already merged on main